client/my-sites/hosting/sftp-card/index.js

```diff
@@ -138,7 +138,7 @@ export const SftpCard = ( {
 	return (
 		<Card className="sftp-card">
 			<MaterialIcon icon="cloud" size={ 32 } />
-			<CardHeading>{ translate( 'SFTP Credentials' ) }</CardHeading>
+			<CardHeading>{ translate( 'SFTP credentials' ) }</CardHeading>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.